modules/codechanges.adoc

```diff
@@ -55,7 +55,7 @@ Username: {{GITLAB_USER}}
 Password: {{GITLAB_PASSWORD}}
 ====
 
-Click the Settings link on the top right of the screen, and then click "Webhooks":
+Click the Settings link on the top right of the screen, and then click "Integrations":

```

Comment:
We've instale latest gitlab and it seems this has changed. 8.16 which by the way installs nicer as requires just anyuid for the serviceaccount for the gitlab pod and not for postgres or redis. 
cc/ @thesteve0 which wrote this part